metabase/metabase
# PopoverWithRef, plus some other stuff

### Description
Migrates `<ClickActionsPopover >` to use Mantine popovers, as well as adds some building blocks for future migrations away from Tippy and Legacy popover components

### How to verify

1. Go to any table, and click the header. It should work as it always does

Current sadness - Clicking the header again moves the popover for a couple cycles before computing the correct spot again.

### Demo
![image](https://github.com/user-attachments/assets/caa2888c-da78-436d-bca6-ffb77c9e2e4d)

### Checklist
